projecttacoma/bulk-submit-demo
# Add button to reset inputs

# Summary
Adds button to reset inputs (exportUrl and selected measure).

## New behavior
The user can now click the button labeled “Reset Inputs.” When clicked, the exportUrl and selected measure will change to their default values (null for selected measure and empty string for exportUrl).

## Code changes
Adds `ResetInputsButton` component, which contains a Mantine Button to reset the input states. 

# Testing guidance
To test, run the following:
* Follow the set-up instructions in the README
* Start up the deqm test server with `npm start`
* Start up the app with `npm run dev`
* Try selecting a measure and typing in the export Url. Then, click on the “Reset Inputs” button and check that both inputs reset.
* Test out the button when only one input is populated - it should be properly reset and nothing should happen to the empty input since it already takes on its default value

Looking for feedback on the following:
* Placement of the button - does it make sense to have the reset button under the input fields? Should it be placed in-line with the input fields?
* Button appearance - size, shape, color, etc.
* What is actually being reset - should we also reset the data requirements state, or does it make sense to keep it on the screen so the user doesn’t have to re-input the measure id and export url if they accidentally push the reset button? Am I over-thinking this?
